### ISSUE TITLE
Version 0.3.7 not working

### ISSUE THREAD

**Issue by zifeo** [koekeishiya/chunkwm#415]
I just upgraded to the latest version through Brew. Usually the accessibility need to be set again and chunkwm then works as normal. However this time, after setting the accessibility, chunkwm does not work (I'm only using the tiling plugin). I've tried multiple times to reinstall and tricks advised in the issues but no change. In the meanwhile, I switch back to kwm. Any idea?

**Comment by koekeishiya** [comment]
did you update your config?

from the changelog:
```
any 'chunkc' command that is run from the config file that is NOT 'chunkc core::<..>' or 'chunkc set ..' MUST be put in the background using &

e.g: chunkc tiling::rule --owner Finder --name Copy --state float &

```

**Comment by sawsc** [comment]
Hi,
I had the same issue and the solution by @koekeishiya solved it.

**Comment by zifeo** [comment]
Thanks for the reactivity, I did follow the changelog. However I was indeed missing one more `&` not yet saved (swap file).
